tests/e2e/framework/kubernetes.go

```diff
@@ -28,31 +29,26 @@ import (
 )
 
 const (
-	yamlSuffix      = ".yaml"
-	mixerHubEnvVar  = "MIXER_HUB"
-	mixerTagEnvVar  = "MIXER_TAG"
-	pilotHubEnvVar  = "PILOT_HUB"
-	pilotTagEnvVar  = "PILOT_TAG"
-	istioInstallDir = "install/kubernetes/templates"
-	istioAddonsDir  = "install/kubernetes/addons"
+	yamlSuffix         = ".yaml"
+	istioInstallDir    = "install/kubernetes"
+	istioAddonsDir     = "install/kubernetes/addons"
+	nonAuthInstallFile = "istio.yaml"
+	authInstallFile    = "istio-auth.yaml"
 )
 
 var (
 	namespace = flag.String("namespace", "", "Namespace to use for testing (empty to create/delete temporary one)")
-	mixerHub  = flag.String("mixer_hub", os.Getenv(mixerHubEnvVar), "Mixer hub")
-	mixerTag  = flag.String("mixer_tag", os.Getenv(mixerTagEnvVar), "Mixer tag")
-	pilotHub  = flag.String("pilot_hub", os.Getenv(pilotHubEnvVar), "Manager hub")
-	pilotTag  = flag.String("pilot_tag", os.Getenv(pilotTagEnvVar), "Manager tag")
+	mixerHub  = flag.String("mixer_hub", "", "Mixer hub, if different from istio.Version")
+	mixerTag  = flag.String("mixer_tag", "", "Mixer tag, if different from istio.Version")
+	pilotHub  = flag.String("pilot_hub", "", "pilot hub, if different from istio.Version")
+	pilotTag  = flag.String("pilot_tag", "", "pilot tag, if different from istio.Version")
 	//caHub        = flag.String("ca_hub", "", "Ca hub")
 	//caTag        = flag.String("ca_tag", "", "Ca tag")
+	authEnable   = flag.Bool("auth_enable", false, "Enable auth")
+	rbacEnable   = flag.Bool("rbac_enable", false, "Enable rbac")
```

Comment:
Do you need those 2 flags ? can you assume rbacEnable when rbac file is specifed ?